### PR TITLE
Basic UI

### PR DESCRIPTION
**Pull request by HashemAbuteen**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "devDependencies": {
+    "tailwindcss": "^3.3.1"
```
Comment (yishain11) [inline client/package.json:43]: why do you use tailwind?

File: client/src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  Route,
+  RouterProvider,
+} from "react-router-dom";
+import Home from "./pages/Home";
+
+// You can do this:
+const router = createBrowserRouter(
+  createRoutesFromElements(
+    <Route path="/" element={<Home />}>
+      {/* <Route path="home" element={<Home />} /> */}
```
Comment (yishain11) [inline client/src/App.jsx:13]: remove commented out lines

File: client/src/components/AddTask.jsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+
+const AddTask = ({ onAddTask }) => {
+  const [taskName, setTaskName] = useState("");
+  const [taskDuration, setTaskDuration] = useState("");
```
Comment (yishain11) [inline client/src/components/AddTask.jsx:5]: duration points to a number, not a string.
if its a date - set a date as default

File: client/src/components/AddTask.jsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        />
+      </div>
+      <div>
+        <label
+          htmlFor="taskDuration"
+          className="block text-sm font-medium text-gray-700"
+        >
+          Task Duration (minutes):
+        </label>
+        <input
```
Comment (yishain11) [inline client/src/components/AddTask.jsx:45]: DRY input. merge with previous input

File: client/package.json (hunk 1 above)
Comment (HashemAbuteen) [reply client/package.json:43]: adds responsive design and structure of pages with classnames only in quick way, I used it before and i liked it

File: client/src/App.jsx (hunk 2 above)
Comment (HashemAbuteen) [reply client/src/App.jsx:13]: done

File: client/src/components/AddTask.jsx (hunk 3 above)
Comment (HashemAbuteen) [reply client/src/components/AddTask.jsx:5]: it is how long the task is going to take (minutes or hours) so i think numbers is good

File: client/src/components/AddTask.jsx (hunk 4 above)
Comment (HashemAbuteen) [reply client/src/components/AddTask.jsx:45]: what do u mean ?

File: client/package.json (hunk 1 above)
Comment (yishain11) [reply client/package.json:43]: so does bootstrap. what is special about tailwind?

**Pull request merged**